src/main/resources/locales/en-US.yml

```
@@ -1079,15 +1079,15 @@ protection:
     TURTLE_EGGS:
       description: "Toggle crushing"
       name: "Turtle Eggs"
-      hint: "Turtle eggs cannot be crushed!"
+      hint: "Turtle egg crushing disabled"
     FROST_WALKER:
       description: "Toggle Frost Walker enchantment"
       name: "Frost Walker"
-      hint: "Frost Walker cannot be used here"
+      hint: "Frost walker disabled"
```

```suggestion
      hint: "Frost Walker disabled"
```
That's an enchantment name, so the uppercases should remain.